recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: 1
+  number: 2
```

Comment:
Could you please bump it again? `master` is now at 2 due to a package corruption issue.